### PR TITLE
Merge sort --> Closes : [#381]

### PR DESCRIPTION
**Pull request by Prerna-07**
closes : #381

### CODE REVIEW

**Comment by Prerna-07** [comment]
@rcoder23 pls review the PR.

File: Topic Wise Questions with Solutions/Linked List/MergeSort LinkedList.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    while curr_node:
+        print(curr_node.data, end = " -> ")
+        curr_node = curr_node.next
+    print('.')
+     
+
+if __name__ == '__main__':                               # Driver Code
+    ll = LLConstruct()
+     
+    n = int(input());
```
Comment (rcoder23) [inline Topic Wise Questions with Solutions/Linked List/MergeSort LinkedList.py:92]: Add  input message

**Review by rcoder23 — CHANGES_REQUESTED** [review]
Add input message . So any one can understand.

**Comment by Prerna-07** [comment]
@rcoder23 ok sir.

**Comment by Prerna-07** [comment]
@rcoder23 done sir.

**Review by rcoder23 — APPROVED** [review]

**Pull request merged**